index.js

```diff
@@ -0,0 +1,110 @@
+
+ let myLibrary = fetch();
+
+let cont = document.getElementById('books');
+let createButton = document.getElementById('create');
+let deleteButton = document.getElementById('books');
+let readBtn = document.getElementById('boxbox');
+
+
+function Book(name, author, numberOfPages) {
+  // the constructor...
+ this.name=name;
+ this.author=author;
+ this.numberOfPages=numberOfPages;
+}
+
+Book.prototype.readStatus= "Not read";
+
+createButton.onclick = function addBookToLibrary() {
+  // do stuff here
+  let name = document.getElementById('orangeForm-name').value
+  let author = document.getElementById('orangeForm-author').value
+  let numberOfPages = document.getElementById('orangeForm-np').value;
+  let newBook = new Book(name, author, numberOfPages);
+  myLibrary.push(newBook);
+  console.log(myLibrary);
+  render(name, author, numberOfPages);
+  
+  save();
+}
+
```

Comment:
- control form values before you submit/push (security)
- remove `console.log`  [see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/console_log_requirement.png)]